### PR TITLE
Algorithm to choose to serialize the observers or not

### PR DESCRIPTION
**Pull request by andreafioraldi**

### CODE REVIEW

File: libafl/src/events/llmp.rs [hunk 1]
```diff
@@ -557,6 +594,27 @@ where
...
     }
 
+    fn serialize_observers<OT>(&mut self, observers: &OT) -> Result<Option<Vec<u8>>, Error>
+    where
+        OT: ObserversTuple<Self::State> + Serialize,
+    {
+        if self.execution_time == Duration::ZERO
+            || self.serialization_time == Duration::ZERO
+            || self.serialization_time + self.deserialization_time < self.execution_time
+            || self.serializations_cnt.trailing_zeros() >= 8
```
Comment (tokatoka) [inline libafl/src/events/llmp.rs:604]: `self.serializations_cnt.trailing_zeros() >= 8`
what does this case mean?

Comment (andreafioraldi) [reply libafl/src/events/llmp.rs:604]: & 0xff, clippy doesn't like bitmasks

File: libafl/Cargo.toml [hunk 2]
```diff
@@ -46,6 +46,9 @@ llmp_compression = ["gzip"] # llmp compression using GZip
 llmp_debug = [] # Enables debug output for LLMP
 llmp_small_maps = [] # reduces initial map size for llmp
 llmp_broker_timeouts = ["std"] # The broker loop will yield occasionally, even without status messages from client nodes
+# Manager features
+count_process_execution = []
```
Comment (domenukk) [inline libafl/Cargo.toml:50]: maybe note what the features do?

File: libafl/Cargo.toml [hunk 3]
```diff
@@ -46,6 +46,9 @@ llmp_compression = ["gzip"] # llmp compression using GZip
...
+no_count_newtestcases = []
```
Comment (domenukk) [inline libafl/Cargo.toml:51]: features shouldn't be negative according to clippy

**Review by domenukk — APPROVED** [review]

File: libafl/Cargo.toml (hunk 3 above)
Comment (domenukk) [reply libafl/Cargo.toml:52]: why not use `count_newtestcases` and invert the branches?

Comment (andreafioraldi) [reply libafl/Cargo.toml:52]: they will never land main

**Pull request merged**